demos/select.html

```diff
@@ -179,37 +180,42 @@ <h2 class="mdc-typography--title">CSS Only</h2>
     <script>
       (function() {
         var MDCSelect = mdc.select.MDCSelect;
-        var root = document.getElementById('js-select');
-        var currentlySelected = document.getElementById('currently-selected');
+        var heroSelect = document.getElementById('hero-js-select');
```

Comment:
Not in demo pages, which don't get transpiled and still need to work in all browsers we support.